publiccode.yml

```diff
@@ -0,0 +1,184 @@
+
+      - Graphical modeler and batch processing
+
+
+      ### 4. Powerful customization and extensibility
+
+
+      - Fully customizable user experience, including user interface and 
+      application settings that cater to power-users and beginners alike
+
+      - Rich [expression
+      engine](https://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/expression.html)
+      for maximum flexibility in visualization and processing
+
+      - Broad and varied [plugin ecosystem](https://plugins.qgis.org/) that
+      includes data connectors, digitizing aids, advanced analysis and charting
+      tools, in-the-field data capture, etc.
+
+      - Style manager for creating, storing, and managing styles
+
+      - [QGIS style hub](https://plugins.qgis.org/styles/) for easy sharing of
+      styles
+
+      - Python and C++ API for standalone (headless) applications as well as
+      in-application comprehensive scripting (PyQGIS)
+
+
+      ### 5. QGIS Server
+
+
+      Headless map server -- running on Linux, macOS, Windows, or in a docker
+      container -- that shares the same code base as QGIS.
+
+
+      - Industry-standard protocols (WMS, WFS, WFS3/OGC API for Features and
+      WCS) allow plug-n-play with any software stack
+
+      - Works with any web server (Apache, nginx, etc) or standalone
+
+      - All beautiful QGIS cartography is supported with best-in-class support
+      for printing
+
+      - Fully customizable with Python scripting support
+    screenshots:
+      - |-
+        https://raw.githubusercontent.com/qgis/QGIS/master/images/README-md/network_analysis_2.png
+    shortDescription: |-
+      QGIS is a full-featured, user-friendly, free-and-open-source geographical
+      information system (GIS) that runs on Unix platforms, Windows and MacOS
+developmentStatus: stable
+intendedAudience:
+  scope:
+    - agriculture
+    - education
+    - emergency-services
+    - energy
+    - environment
+    - government
+    - infrastructures
+    - local-authorities
+    - transportation
+    - tourism
+    - welfare
+it:
+  conforme:
+    gdpr: false
+    lineeGuidaDesign: false
+    misureMinimeSicurezza: false
+    modelloInteroperabilita: false
+  countryExtensionVersion: '0.2'
+  piattaforme:
+    anpr: false
+    cie: false
+    pagopa: false
+    spid: false
+landingURL: 'https://qgis.org/funding'
+legal:
+  license: LGPL-2.0-only
+localisation:
+  availableLanguages:
+    - en
+    - it
+    - es
+    - fr
+    - de
+  localisationReady: true
+logo: |-
+  https://raw.githubusercontent.com/qgis/QGIS/master/images/README-md/main_logo.png
+maintenance:
+  contacts:
+    - email: qgis-community-team@lists.osgeo.org
+      name: QGIS developers
+  type: community
+name: QGIS
+platforms:
+  - linux
+  - windows
+  - mac
+releaseDate: '2021-05-14'
+roadmap: 'https://www.qgis.org/en/site/getinvolved/development/roadmap.html'
+softwareType: standalone/desktop
+softwareVersion: 3.18.3
```

Comment:
That and the above releaseDate will get out of sync quickly. Either there should be some process to update this file, or the release procedure should be updated. CC @jef-n

Reply:
> That and the above releaseDate will get out of sync quickly. Either there should be some process to update this file, or the release procedure should be updated. CC @jef-n

similar to `linux/org.qgis.qgis.appdata.xml.in` (in scripts/release.pl).   Are there other standards?

Reply:
@rouault as @libremente said you can keep releaseDate and softwareVersion in sync using this GH action https://github.com/italia/publiccode-softwareversion-check-action . Let me know if you need more details

Reply:
someone has to put in place this github action in this repository

Reply:
I'd add that to release.pl - just like the update to `linux/org.qgis.qgis.appdata.xml.in` - which to some extend has the same information.

Reply:
If this is really to be done through github actions, I'd say this should be pushed directly using something like https://github.com/EndBug/add-and-commit, I'm not a bit fan of pushing the maintenance burden of this monthly PR to our review team.
What should happen to this file on `release-x` branches?
Are these fields mandatory?

Reply:
@m-kuhn publiccode.yml should be placed in the root of the repo and updated in the main branch, so that our crawler can fetch and read it. Our GH action (https://github.com/italia/publiccode-softwareversion-check-action) only checks if a new tag is released and if it matches the `softwareVersion` specified in publiccode.yml. If the latest tag is not equal to the `softwareVersion` value, our GH action will change it and then you can attach another action to decide what to do (open a PR or Add-and-commit)

Reply:
We have multiple parallel releases each month (in June this will be 3.20.0 and 3.16.8).

Reply:
Ok @m-kuhn following publiccode.yml standard, looks like `softwareVersion` is not a mandatory field,  we can remove it (@mfortini will update this PR accordingly). You can still consider to use our GH action (or whatever you want) to update `releaseDate` field automatically (this will be equal to the latest release date).

Reply:
Thanks for clarification :+1: 

Does this mean `releaseDate` *is* a mandatory field?
I'm just trying to see if we can trim this to a 0 maintenance overhead solution here (if there is such a thing)

Reply:
@m-kuhn yes `releaseDate` is mandatory in the current standard, unless the project is still in its pre-release state.
https://github.com/publiccodeyml/publiccode.yml/blob/master/docs/en/schema.core.rst#key-releasedate

Reply:
> We have multiple parallel releases each month (in June this will be 3.20.0 and 3.16.8).

Why would that be an issue?   The release date and version would be in the respective branch and master itself is never released.

Reply:
@jef-n see https://github.com/qgis/QGIS/pull/43388#discussion_r639806361

> publiccode.yml should be placed in the root of the repo and updated in the main branch